content/en/docs/tasks/debug/debug-cluster/flow-control.md

```diff
@@ -0,0 +1,178 @@
+  workload-low,      service-accounts, 14,         0,                   system:serviceaccount:default:loadtest, 2023-07-18T00:13:08.986534842Z, 10,           0,          0s,                0001-01-01T00:00:00Z,           system:serviceaccount:default:loadtest, list,   /api/v1/namespaces/kube-stress/configmaps, kube-stress, ,       v1,         configmaps,
+  workload-low,      service-accounts, 14,         1,                   system:serviceaccount:default:loadtest, 2023-07-18T00:13:09.086476021Z, 10,           0,          0s,                0001-01-01T00:00:00Z,           system:serviceaccount:default:loadtest, list,   /api/v1/namespaces/kube-stress/configmaps, kube-stress, ,       v1,         configmaps,
+  ```
+
+  Explanation for selected column names:
+  - `QueueIndex`: The index of the queue. It will be -1 for priority levels
+    without queues.
+  - `RequestIndexInQueue`: The index in the queue for a given request. It will
+    be -1 for executing requests.
+  - `InitialSeats`: The number of seats will be occupied during the initial
+    (normal) stage of execution of the request.
+  - `FinalSeats`: The number of seats will be occupied during the final stage
+    of request execution, accounting for the associated WATCH notifications.
+  - `AdditionalLatency`: The extra time taken during the final stage of request
+    execution. FinalSeats will be occupied during this time period. It does not
+    mean any latency that a user will observe.
+  - `StartTime`: The time a request starts to execute. It will be
+    0001-01-01T00:00:00Z for queued requests.
+
+## Debug logging
+
+At `-v=3` or more verbose the server outputs an httplog line for every
+request, and it includes the following attributes.
+
+- `apf_fs`: the name of the flow schema to which the request was classified.
+- `apf_pl`: the name of the priority level for that flow schema.
+- `apf_iseats`: the number of seats determined for the initial
+  (normal) stage of execution of the request.
+- `apf_fseats`: the number of seats determined for the final stage of
+  execution (accounting for the associated WATCH notifications) of the
+  request.
+- `apf_additionalLatency`: the duration of the final stage of
+  execution of the request.
+
+At higher levels of verbosity there will be log lines exposing details
+of how APF handled the request, primarily for debugging purposes.
+
+## Response headers
+
+APF adds the following two headers to each HTTP response message.
+
+- `X-Kubernetes-PF-FlowSchema-UID` holds the UID of the FlowSchema
+  object to which the corresponding request was classified.
+- `X-Kubernetes-PF-PriorityLevel-UID` holds the UID of the
+  PriorityLevelConfiguration object associated with that FlowSchema.
+
```

Comment:
I would mention whether these appear in audit logs. AIUI they don't, and you can't enable that - have I got this right?

Reply:
Updated.
Will not appear in audit log.